maxfw/core/api.py

```diff
@@ -20,6 +20,12 @@ def get(self):
         """To be implemented"""
         raise NotImplementedError()
 
+class LabelsAPI(MAXAPI):
```

Comment:
Maybe this should be more generic, like `CustomMAXAPI`

Reply:
I was actually thinking about that but a comment by @ptitzler made me use this. He made a comment along the line of: "any model that classifies thing based on a list of labels should have a `labels` endpoint". Based on that I thought a specific endpoint like this made sense. Otherwise we could just rename `MetadataAPI` tp `GetAPI` and `PredictAPI` to `PostAPI` and that would make this whole file more generic.

Reply:
The idea here is every MAX model _must_ implement `MetadataAPI` and `PredictAPI`. `LabelsAPI` is common but not applicable to every model. What value does it add to have it here in `maxfw`?

Reply:
I agree with Nick here, which is why I think we should have a `CustomMAXAPI` to cover things that are more specific to a given model.

Reply:
@MLnick 's comment clarified the issue for me, given `LabelsAPI` is not required for every model it doesn't need to be here. Based on this `ModelLabelsAPI() can just extend `MAX_API` directly in Object Detector and this code chunk can be removed. I'll test and update my PRs accordingly. And to clarify why not make a `CustomMAXAPI`, that's essentially what `MAX_API` already does now, based on @MLnick 's comment, the purpose of `MetadataAPI` and `PredictAPI` are to "force" dev's to extend them.

Reply:
I'm not sure if `MAX_API` should be used by default. I realize there's not much in these classes to begin with but in the future we may want to make a change to these custom APIs without affecting the rest of the APIs.

Reply:
actually I was mixing up `MAX_API` and `MAXAPI` in my previous comment

Reply:
I meant that for MAXAPI too

Reply:
I guess my question then is what would the difference between your purposed `CustomMAXAPI` and the current `MAXAPI` be?

Reply:
`CustomMAXAPI` would extend `MAXAPI`. If we wanted to update all APIs for some reason we'd change `MAXAPI`. If we wanted to push an update to affect all Custom APIs we'd make a change in `CustomMAXAPI`.

Reply:
Ok I see, I'll make the code update then